single_lwr_example/single_lwr_moveit/launch/moveit_planning_execution.launch

```diff
@@ -0,0 +1,78 @@
+<launch>
+  <!-- The planning and execution components of MoveIt! configured to run -->
+  <!-- using the ROS-Industrial interface. -->
+ 
+  <!-- Non-standard joint names:
+       - Create a file [robot_moveit_config]/config/joint_names.yaml
+           controller_joint_names: [joint_1, joint_2, ... joint_N] 
+       - Update with joint names for your robot (in order expected by rbt controller)
+       - and uncomment the following line: -->
+  <!-- <rosparam command="load" file="$(find [robot_moveit_config])/config/joint_names.yaml"/> -->
+ 
+  <!-- the "sim" argument controls whether we connect to a Simulated or Real robot -->
+  <!--  - if sim=false, a robot_ip argument is required -->
+  <arg name="sim" default="true" doc="If true, the robot will be simulated in Gazebo" />
+  <arg name="robot_ip" default="192.168.0.20" unless="$(arg sim)" doc="The IP address of the robot" />
+  <arg name="robot_port" default="49939" unless="$(arg sim)" doc="The listening port of the FRI interface" />
+  <arg name="rviz" default="true" doc="Launch rviz in order to command the robot" />
+  <!--<arg name="impedance" default="true" doc="Use impedance control" />-->
+ 
+  <!-- load the robot_description parameter before launching ROS-I nodes -->
+  <include file="$(find single_lwr_moveit)/launch/planning_context.launch" >
+    <arg name="load_robot_description" value="true" />
+  </include>
+
+	<!-- Load joint controller configurations from YAML file to parameter server -->
+	<rosparam file="$(find single_lwr_robot)/config/controllers.yaml" command="load"/>
+	
+    <node ns="lwr" name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
+        output="screen"  args="joint_state_controller joint_trajectory_controller"/> 
+	<!--<node unless="$(arg impedance)" ns="lwr" name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
+		output="screen"  args="joint_state_controller position_joint_trajectory_controller"/> 
+    <node if="$(arg impedance)" ns="lwr" name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
+        output="screen"  args="joint_state_controller joint_trajectory_controller"/> -->
+  
+  <remap from="/lwr/joint_states" to="/joint_states"/>  <!--TODO: why is this needed??-->
```

Comment:
Because the robot_state_publisher subscribes to /joint_states, where nothing will be published in this case. A more general way to do it is:
``` xml
<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
  <param name="use_gui" value="$(arg gui)"/>
  <rosparam param="source_list">[/lwr/joint_states]</rosparam>
</node>
```

So, in case you have more devices, you can put them all there and have a single /joint_states with all devices. Check [this one](https://github.com/CentroEPiaggio/vito-robot/blob/master/vito_description/launch/display.launch#L35-L39)